src/store/plugins/fungibleTokens.js

```diff
@@ -4,6 +4,7 @@ import BigNumber from 'bignumber.js';
 import {
   unionBy, isEqual, isEmpty, uniqBy,
 } from 'lodash-es';
+import * as routerInterface from 'dex-contracts-v2/build/IAedexV2Pair.aes';
```

Comment:
It is not actually a routerInterface it is a pairInterface. Can be imported like that.
```suggestion
import pairInterface from 'dex-contracts-v2/build/IAedexV2Pair.aes';
```